packages/adapter/addon/rest.js

```diff
@@ -30,6 +30,8 @@ import { warn } from '@ember/debug';
 import { DEBUG } from '@glimmer/env';
 
 const Promise = EmberPromise;
+const jQueryDescriptor = Object.getOwnPropertyDescriptor(Ember, '$');
+const jQuery = jQueryDescriptor ? jQueryDescriptor.value || jQueryDescriptor.get() : undefined;
```

Comment:
Hmm... I don't think is right. Calling `jQueryDescriptor.get` will trigger the deprecation warning.

Reply:
@wagenet I confirmed that it does not, and also it's the only way to get the value in 3.9+

Reply:
For the `useFetch` check you just want to check if it would exist right? Then if it actually gets used, triggering the deprecation warning would be correct.

Reply:
<img width="380" alt="image" src="https://user-images.githubusercontent.com/9835/56441382-ba844c80-62a1-11e9-8d23-2f009928c344.png">

Reply:
```suggestion
const jQueryDefined = jQueryDescriptor ? jQueryDescriptor.value || jQueryDescriptor.get : false;
```

Reply:
@wagenet the deprecation for `ajax` would be incorrect because you may have actually installed `@ember/jquery`.

I am surprised you receive that assertion there as testing locally I do not get that. What version of Ember are you using?

Reply:
That's 3.9.1, in the test console.

If I'm understanding correctly, if ember-jquery is installed, using `Ember.$` still isn't correct right? So either way, we'd need a different code path to avoid the deprecation.

Reply:
The deprecation warning is in the `get` for `Ember.$` so I don't see any way to avoid the deprecation warning there.

Reply:
> If I'm understanding correctly, if ember-jquery is installed, using Ember.$ still isn't correct right? So either way, we'd need a different code path to avoid the deprecation.

Correct.

Reply:
@rwjblue do you know why ember changed the descriptor in 3.9 to not have `value` and use `get` instead?

Reply:
@wagenet I'm confused still why 3.9 for me did not print a deprecation when I tested `descriptor.get()` (maybe it was 3.9.0?, maybe we had a release with the deprecation broken? maybe I trolled myself and it doesn't print more than once and it already had?)

Reply:
https://github.com/emberjs/ember.js/commit/3c00987af16c2e80c81895f6e18cfe10db37387f

Reply:
Looks the same in [3.9.0](https://github.com/emberjs/ember.js/blob/v3.9.0/packages/ember/index.js#L564) and [3.9.1](https://github.com/emberjs/ember.js/blob/v3.9.1/packages/ember/index.js#L564)

Reply:
I moved to using global jQuery